### PR TITLE
Improve code style to keep consistency across app and with Android guidelines

### PR DESCRIPTION
**Pull request by FarbodSalamat-Zadeh**
Here are some improvements I've made so far to improve code style across the app.

### CODE REVIEW

File: PowerUp/app/src/main/java/powerup/systers/com/AvatarActivity.java [hunk 1]
```diff
@@ -129,16 +132,21 @@ public void onCreate(Bundle savedInstanceState) {
...
+    public void onClick(View v) {
+        switch (v.getId()) {
+            case R.id.continueButton:
                 if (mFromActivity == 1) {
-                    SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(AvatarActivity.this);
-                    boolean hasPreviouslyStarted = prefs.getBoolean(getString(R.string.preferences_has_previously_started), false);
+                    SharedPreferences prefs =
+                            PreferenceManager.getDefaultSharedPreferences(AvatarActivity.this);
+                    boolean hasPreviouslyStarted = prefs.getBoolean(
+                            getString(R.string.preferences_has_previously_started), false);
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/java/powerup/systers/com/AvatarActivity.java]: Remove extra line

**Comment by FarbodSalamat-Zadeh** [comment]
Although I've finished the GCU task, I will still make some additional improvements regarding code style to improve the pull request.

**Comment by chhavip** [comment]
I hope the GCI task was approved, unfortunately we wont be able to merge this PR as each of the suggestions will need its own PR covering the entire application and not just some files plus this also has conflicts.

**Comment by FarbodSalamat-Zadeh** [comment]
@chhavip

Yes, the GCI task was approved.

In that case, I'll close this PR - thanks for letting me know.

**Pull request closed without merging**